packages/react/src/VideoPlayer/VideoPlayer.module.css

```diff
@@ -18,20 +18,37 @@
 
 .VideoPlayer__container {
   width: 100%;
-  position: relative;
   border-radius: var(--brand-borderRadius-medium);
+  overflow: hidden;
+  background: #000000;
```

Comment:
Is this supposed to be black in all color modes? or the usual background color?

If former you want:

```suggestion
  background-color: var(--base-color-scale-black-0);
```

Reply:
Thanks! Wasn't sure what to set these to, as per my comment in the description

Reply:
@rezrah I've just spotted that this token isn't pure black (#1f2328).

My personal preference would be to use pure black here. Would you like me to stick with the token, or can I use #000000?

Reply:
No it looks ok, if sticking to the token is preferable then we can go with that 👍

Both options below for transparency:

| Black | Off-black (design token) |
| --- | --- |
| ![Visual-Comparison-VideoPlayer-VideoPlayer-Without-Branding-1-linux](https://github.com/user-attachments/assets/2470a82c-244c-403f-a2f4-64e5a078c25b) | ![Visual-Comparison-VideoPlayer-VideoPlayer-Without-Branding-1-linux](https://github.com/user-attachments/assets/04a6400d-9989-4cd1-a04f-dd2ef3d2caad) |

Reply:
Ah sorry, that's not _exactly_ right. The title overlay will need a pure black background to guarantee contrast with the content. The rest of it can be off-black though.

Reply:
🤔 okay, sounds like you need a fixed `#000` value if it's a contrast issue. For context, at some point we need to introduce our primer css lint rules, and hexes will be flagged, so trying to reduce ones we introduce if we can avoid it. On this occasion, let's stick to hex with an annotation for future reference? Separately, we can look at making `black-0` an actual black.